src/egl/drivers/dri2/egl_dri2.h

```diff
@@ -314,6 +314,7 @@ struct dri2_egl_surface
    struct ANativeWindowBuffer *buffer;
    __DRIimage *dri_image_back;
    __DRIimage *dri_image_front;
+   size_t override_max_buffer_count;
```

Comment:
Dont forget to fix u commit message before sending it to review to mesa mailing list.